### PR TITLE
release: sourcegraph@3.35.0

### PR DESCRIPTION
**Pull request by caugustus-sourcegraph**
This pull request is part of the Sourcegraph 3.35.0 release.

* [Release batch change](https://k8s.sgdev.org/organizations/sourcegraph/batch-changes/release-sourcegraph-3.35.0)
* [Tracking issue](https://github.com/sourcegraph/sourcegraph/issues/28613)

### :warning: Additional changes required

These steps must be completed before this PR can be merged, unless otherwise stated. Push any required changes directly to this PR branch.

- [x] Update the upgrade guides in `doc/admin/updates`
- [x] Ensure all other pull requests in the batch change have been merged - then run `yarn run release release:finalize` to generate the tags required, re-run Buildkite on this branch, and ensure the build passes before merging this pull request

cc @kevinwojo

### CODE REVIEW

File: doc/admin/updates/kubernetes.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ and any manual migration steps you must perform.
 
 ## 3.34 -> 3.35
 
+The `query-runner` service has been decomissioned in the 3.35.0 release. You can safely remove the `query-runner` service from your installation.
+
+*How smooth was this upgrade process for you? You can give us your feedback on this upgrade by filling out [this feedback form](https://share.hsforms.com/1aGeG7ALQQEGO6zyfauIiCA1n7ku?update_version=3.34).*
+
+## 3.34 -> 3.35
+
 The query-runner deployment has been removed, so if you deploy with a method other than the `kubectl-apply-all.sh`, a manual removal of the deployment may be necessary.
```
Comment (caugustus-sourcegraph) [inline doc/admin/updates/kubernetes.md:21]: Let's use this text (and remove the duplicate `## 3.34 -> 3.35` section instead.

File: doc/admin/updates/docker_compose.md [hunk 2]
```diff
@@ -10,6 +10,12 @@ Each section comprehensively describes the steps needed to upgrade, and any manu
 
 <!-- GENERATE UPGRADE GUIDE ON RELEASE (release tooling uses this to add entries) -->
 
+## 3.34 -> 3.35
+
+The `query-runner` service has been decomissioned in the 3.35.0 release. You can safely remove the `query-runner` service from your installation.
```
Comment (caugustus-sourcegraph) [inline doc/admin/updates/docker_compose.md:15]: docker-compose installations don't need to do anything special so maybe something like:
```suggestion
The `query-runner` service has been decommissioned in the 3.35.0 release and will be removed during the upgrade.
```

File: doc/admin/updates/kubernetes.md (hunk 1 above)
Comment (kevinwojo) [reply doc/admin/updates/kubernetes.md:21]: Yeah I just saw that 😮‍💨

File: doc/admin/updates/docker_compose.md [hunk 3]
```diff
@@ -10,6 +10,12 @@ Each section comprehensively describes the steps needed to upgrade, and any manu
 
 <!-- GENERATE UPGRADE GUIDE ON RELEASE (release tooling uses this to add entries) -->
 
+## 3.34 -> 3.35
+
+The `query-runner` service has been decommissioned in the 3.35.0 release and will be removed during the upgrade.
```
Comment (caugustus-sourcegraph) [inline doc/admin/updates/docker_compose.md:15]: I'm a big jerk and didn't confirm what I was saying.

The docker-compose command needs to be run with `--remove-orphans` in order to delete query-runner.

Comment (caugustus-sourcegraph) [reply doc/admin/updates/docker_compose.md:15]: We do mention that in the upgrade docs but it's good to call out. Also, probably need to preserve the `follow the [standard upgrade procedure](../install/docker-compose/operations.md#upgrade) to upgrade your deployment` bit here.

File: doc/admin/updates/kubernetes.md (hunk 1 above)
Comment (caugustus-sourcegraph) [reply doc/admin/updates/kubernetes.md:21]: Ah, and also include ` follow the [standard upgrade procedure](../install/kubernetes/update.md) to upgrade your deployment.`

File: doc/admin/updates/kubernetes.md [hunk 4]
```diff
@@ -14,6 +14,8 @@ and any manual migration steps you must perform.
 
 The query-runner deployment has been removed, so if you deploy with a method other than the `kubectl-apply-all.sh`, a manual removal of the deployment may be necessary.
 
+*How smooth was this upgrade process for you? You can give us your feedback on this upgrade by filling out [this feedback form](https://share.hsforms.com/1aGeG7ALQQEGO6zyfauIiCA1n7ku?update_version=3.34).*
```
Comment (caugustus-sourcegraph) [inline doc/admin/updates/kubernetes.md:17]: If you add `Follow the [standard upgrade procedure](../install/kubernetes/update.md) to upgrade your deployment.` above this, I think we're good!

**Review by kevinwojo — APPROVED** [review]

**Pull request merged**